### PR TITLE
Setup Github actions workflow in the backend

### PR DESCRIPTION
**Pull request by Ogenbertrand**

### CODE REVIEW

**Review by Koufan-De-King — APPROVED** [review]
All good.

**Review by bryandino673 — APPROVED** [review]

**Review by kouamschekina — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Build with Gradle
+        run: ./gradlew build
+
+      - name: Run Test
+        run: ./gradlew test
+
+      - name: Upload build artifacts
```
Comment (stephane-segning) [inline .github/workflows/build-test.yml:37]: Why upload this?

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (Ogenbertrand) [reply .github/workflows/build-test.yml:37]: @stephane-segning 
To upload the artifact generated during a workflow run sir.

**Comment by Ogenbertrand** [comment]
@stephane-segning 
[modify:buid-test.yml](https://github.com/stephane-segning/lynx-scanner-backend/pull/18/commits/b10efeab8d6ad81118a9d98aeab22e299279054a)

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**